convene-web/config/routes.rb

```diff
@@ -22,6 +22,9 @@
     end
   end
 
+  resources :space_memberships do
```

Comment:
Before we merge, we will probably want to rename this to `:invitations` and restrict it to only the`create` and destroy actions.

See an example of a resource being restricted to only particular actions here: https://guides.rubyonrails.org/routing.html#configuring-the-rails-router